add-ons/ansible-service-broker/ansible-service-broker.addon

```diff
@@ -1,12 +1,14 @@
 # Name: ansible-service-broker
 # Description: Deploys the Ansible Service Broker
-# Required-vars:
+# Required-vars: BROKER_REPO_TAG, APBTOOLS_REPO_TAG, DOCKERHUB_ORG
+# Var-Defaults: BROKER_REPO_TAG=ansible-service-broker-1.1.8-1, APBTOOLS_REPO_TAG=apb-1.1.6-1, DOCKERHUB_ORG=ansibleplaybookbundle
```

Comment:
We must define OpenShift-Version metadata as >=3.7.1. Refer https://docs.openshift.org/latest/minishift/using/addons.html#addon-openshift-version-semantics